libcxx/test/std/containers/sequences/vector/common.h

```diff
@@ -125,4 +220,39 @@ inline void check_new_delete_called() {
   assert(globalMemCounter.aligned_new_array_called == globalMemCounter.aligned_delete_array_called);
 }
 
+class Rnd {
```

Comment:
Let's move the functions outside of the class since this isn't related to randomness anymore.